### PR TITLE
Tags redesign

### PR DESCRIPTION
**Pull request by chloe0592**
Updating `Tag` component with new design tokens

### CODE REVIEW

**Review by FilPob — APPROVED** [review]

File: packages/components/tag/src/tag-body.tsx [hunk 1]
```diff
@@ -45,16 +46,17 @@ const getClickableContentWrapperStyles = (type: TTagBodyProps['type']) => {
 
 const getTextDetailColor = (isDisabled: TTagBodyProps['isDisabled']) => {
   if (isDisabled) return designTokens.fontColorForTagWhenDisabled;
-  return designTokens.fontColorForTag;
+  return designTokens.colorSolid;
 };
 
-const getContentWrapperStyles = (props: TTagBodyProps) => {
+const GetContentWrapperStyles = (props: TTagBodyProps) => {
```
Comment (CarlosCortizasCT) [inline packages/components/tag/src/tag-body.tsx:52]: I don't think we should use capital letters for the first one since this is not a component.

**Review by CarlosCortizasCT — COMMENTED** [review]
This is looking good 👍 

I've left some comment with some ideas for improvement.

File: packages/components/tag/src/tag.tsx [hunk 2]
```diff
@@ -109,37 +110,39 @@ const Tag = (props: TTagProps) => {
...
+                    fill: ${designTokens.fontColorForTagRemoveIconWhenHovered} !important;
                   }
                 }
                 > svg * {
-                  fill: ${designTokens.fontColorForTag};
+                  fill: ${designTokens.fontColorForTagRemoveIcon} !important;
                 }
                 &:disabled > svg * {
-                  fill: ${designTokens.fontColorForTagWhenDisabled};
+                  fill: ${designTokens.fontColorForTagWhenDisabled} !important;
```
[the remark this answers is not in the document]
Comment (Rhotimee) [reply packages/components/tag/src/tag.tsx]: Updated here -> [458a96d](https://github.com/commercetools/ui-kit/pull/2375/commits/458a96da84db6a22023610675609c1d1d9339638)

File: packages/components/tag/src/tag-body.tsx (hunk 1 above)
Comment (Rhotimee) [reply packages/components/tag/src/tag-body.tsx]: updated here -> [89a39d0](https://github.com/commercetools/ui-kit/pull/2375/commits/89a39d018f49c9437d2b5ff3e70f76e0c4fb75a7)

File: packages/components/tag/src/tag-body.tsx [hunk 3]
```diff
@@ -119,7 +130,23 @@ const TagBody = (props: TTagBodyProps) => {
     >
       <Spacings.Inline scale="s" alignItems="center">
         {props.isDraggable && !props.isDisabled ? (
-          <DragIcon data-testid="drag-icon" size="medium" />
+          <AccessibleButton
```
[the remark this answers is not in the document]
Comment (Rhotimee) [reply packages/components/tag/src/tag-body.tsx]: updated [458a96d](https://github.com/commercetools/ui-kit/pull/2375/commits/458a96da84db6a22023610675609c1d1d9339638)

File: design-system/materials/internals/definition.yaml [hunk 4]
```diff
@@ -74,6 +74,7 @@ choiceGroupsByTheme:
         color-solid-05: 'hsl(0deg 0% 10% / 5%)'
         color-solid-10: 'hsl(0deg 0% 10% / 10%)'
         color-surface: '#fff'
+        color-inherit: 'inherit'
```
[the remark this answers is not in the document]
Comment (Rhotimee) [reply design-system/materials/internals/definition.yaml]: Updated -> [2a0f15c](https://github.com/commercetools/ui-kit/pull/2375/commits/2a0f15ce1f2a2fed4401d8ad0485e2b1ba09b4b2)

File: packages/components/icons/src/generated/AngleDownReact.tsx [hunk 5]
```diff
@@ -133,29 +113,24 @@ export const getIconStyles = (props: Props) => css`
...
   flex-shrink: 0;
 `;
-
 const SvgAngleDown = (props: SVGProps) => (
   <svg
     width={24}
     height={24}
     viewBox="0 0 24 24"
     xmlns="http://www.w3.org/2000/svg"
-    role="img"
```
Comment (CarlosCortizasCT) [inline packages/components/icons/src/generated/AngleDownReact.tsx:143]: I think icons should not have changes in this PR.
Perhaps you have an outdated version of `@svgr/cli` installed?
It should be version `6.5.1`.

Comment (Rhotimee) [reply packages/components/icons/src/generated/AngleDownReact.tsx:143]: I'm using the `@svgr/cli` version 6.5.1. 
I'm not sure why there are changes still

File: design-system/materials/internals/definition.yaml [hunk 6]
```diff
@@ -576,6 +584,15 @@ decisionGroupsByTheme:
         font-color-for-tag:
           description: ''
           choice: color-solid
+        font-color-for-tag-remove-icon:
+          description: ''
```
[the remark this answers is not in the document]
Comment (Rhotimee) [reply design-system/materials/internals/definition.yaml]: Updated [94b9583](https://github.com/commercetools/ui-kit/pull/2375/commits/94b95831ac000ee04362733d3643da23ef7a21c5)

File: packages/components/icons/src/generated/AngleDownReact.tsx (hunk 5 above)
Comment (Rhotimee) [reply packages/components/icons/src/generated/AngleDownReact.tsx]: Here is the commit -> [5602bb9](https://github.com/commercetools/ui-kit/pull/2375/commits/5602bb924cbebb9ea9f23beee9db384637fb4f9d)

**Review by CarlosCortizasCT — APPROVED** [review]
Great job! 👏

**Pull request merged**